src/main/java/com/google/devtools/build/lib/rules/cpp/proto/CcProtoAspect.java

```diff
@@ -256,6 +256,7 @@ private CcCompilationHelper initializeCompilationHelper(
       }
 
       helper.addDeps(ruleContext.getPrerequisites("deps", TARGET));
+      helper.setCopts(ImmutableList.of("-w"));
```

Comment:
This won't work: this assumes that the compiled is gcc/clang and leaves MSVC users out in the cold.

Reply:
Thanks for the review! I'll try it on Windows just to be on the safe side. 
The MSVC docs state [`Options are specified by either a forward slash (/) or a dash (–).`](https://msdn.microsoft.com/en-us/library/610ecb4h.aspx) (see the `option` entry). This in combination with [the Warning levels page](https://docs.microsoft.com/en-gb/cpp/build/reference/compiler-option-warning-level) leads me to believe that it _should_ work. But then again, this is Windows we're talking about.

Reply:
MSVC can accept `-w` to disable all warnings just fine, but if someone *append* `-W3`/`-W4` etc. *after* `-w`, you will get [D9025` warning](https://msdn.microsoft.com/en-us/library/8k3f51f1.aspx) that cannot be silenced at all, causing a waterfall of warnings.